src/test/java/com/knightcharacter/app/integration/SkillResourceIntegrationTest.java

```diff
@@ -0,0 +1,365 @@
+import static com.knightcharacter.app.TestConstants.buildJsonPathToNameInBonusListByIndex;
+import static com.knightcharacter.app.TestConstants.buildJsonPathToNameInBonusListNestedInSkillListByIndexes;
+import static com.knightcharacter.app.TestConstants.buildJsonPathToNameInListByIndex;
+import static com.knightcharacter.app.TestConstants.buildJsonPathToRarityInBonusListByIndex;
+import static com.knightcharacter.app.TestConstants.buildJsonPathToRarityInBonusListNestedInSkillListByIndexes;
+import static com.knightcharacter.app.TestConstants.buildJsonPathToSkillBoostInBonusListByIndex;
+import static com.knightcharacter.app.TestConstants.buildJsonPathToSkillBoostInBonusListNestedInSkillListByIndexes;
+import static com.knightcharacter.app.TestConstants.buildPutSkillByIdUrl;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.delete;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.put;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import com.knightcharacter.app.factories.BonusFactory;
+import com.knightcharacter.app.factories.SkillFactory;
+import com.knightcharacter.app.gateway.privatedb.repository.BonusRepository;
+import com.knightcharacter.app.gateway.privatedb.repository.SkillRepository;
+import com.knightcharacter.app.gateway.privatedb.representation.Bonus;
+import com.knightcharacter.app.gateway.privatedb.representation.Skill;
+import com.knightcharacter.app.rest.request.SkillRequestDto;
+
+import java.util.List;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.transaction.annotation.Transactional;
+
+@SpringBootTest
+@AutoConfigureMockMvc
+class SkillResourceIntegrationTest {
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @Autowired
+    private ObjectMapper objectMapper;
+
+    @Autowired
+    private SkillRepository skillRepository;
+
+    @Autowired
+    private BonusRepository bonusRepository;
+
+    @AfterEach
+    void tearDown() {
+        skillRepository.deleteAll();
+        bonusRepository.deleteAll();
+    }
+
+    @Test
+    @Transactional
+    void addSkill_shouldAddSkillAndReturnIt_whenRequestIsCorrect() throws Exception {
```

Comment:
It should return not the skill but the response if to make it logically right.

Reply:
ok, so we need to rename all same tests in all microservices, because the naming is same everywhere

Reply:
Yeah, I think so. Or at least you can change it here and create a task for renaming them all. Of course, if you agree with this suggestion, maybe you think that there is nothing wrong with it or that it's better to not to be troubled about it to much

Reply:
Ok, I agree